freud/common.py

```diff
@@ -44,11 +44,11 @@ def convert_box(box):
     .. moduleauthor:: Bradley Dice <bdice@bradleydice.com>
 
     Args:
-      box (box-like object (see :meth:`freud.box.Box.from_box`)): Box to
-          check and convert if needed.
+        box (box-like object (see :meth:`freud.box.Box.from_box`)): Box to
+            check and convert if needed.
 
     Returns:
-      py:class:`freud.box.Box`: freud box.
+        :class:`freud.box.Box`: freud box.
```

Comment:
Just fixing some docstrings here - they're not rendered but this makes it consistent with everywhere else in freud (to my knowledge).